Source/WebGPU/WGSL/WGSLShaderModule.h

```diff
@@ -47,9 +47,10 @@ class ShaderModule {
         : ShaderModule(source, { })
     { }
     
-    ShaderModule(const String& source, const Configuration& configuration)
+    ShaderModule(const String& source, const Configuration& configuration, HashSet<String>&& supportedFeatures = { })
```

Comment:
Any reason not to put it in the configuration?

Reply:
Oh no, I will move it there. Thanks for the suggestion